### PR TITLE
[wolfram/zh-cn]Create wolfram-cn.html.markdown

### PR DESCRIPTION
**Pull request by wuyudi**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: zh-cn/wolfram-cn.html.markdown [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+Wolfram 语言有几个界面。
+
+- Raspberry Pi 上的命令行内核界面（就叫 _The Wolfram Language_），它是交互式运行的，不能产生图形输入。
+- _Mathematica_ 是一个丰富的文本/数学编辑器，内置交互式的 Wolfram: 在 "代码单元 "上按 shift+Return 键可以创建一个输出单元，输出结果，这不是动态的。
+- _Wolfram Workbench_，这是 Eclipse 与 Wolfram 语言后端的界面。
+
+本例中的代码可以在任何界面中输入，并使用 Wolfram Workbench 进行编辑。直接加载到 Mathematica 中可能会很不方便，因为该文件不包含单元格格式化信息（这将使该文件作为文本阅读时变得一团糟）--它可以被查看/编辑，但可能需要一些设置。
+
+```mma
+(* 这是一个评论 *)
```
Comment (imba-tjd) [inline zh-cn/wolfram-cn.html.markdown:22]:
```suggestion
(* 这是一个注释 *)
```

File: zh-cn/wolfram-cn.html.markdown [hunk 2]
```diff
@@ -0,0 +1,140 @@
+---
+language: wolfram
+contributors:
+  - ["hyphz", "http://github.com/hyphz/"]
+translators:
+  - ["wuyudi", "http://github.com/wuyudi/"]
+filename: learnwolfram.nb
```
Comment (imba-tjd) [inline zh-cn/wolfram-cn.html.markdown:7]:
```suggestion
filename: learnwolfram-cn.nb
```

File: zh-cn/wolfram-cn.html.markdown [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+Append[myList, 5]         (* {1, 2, 3, 4, 5} - 注意 myList 没有更新 *)
+Prepend[myList, 5]        (* {5, 1, 2, 3, 4} - 如果想让 myList 更新，就加上 "myList = " *)
+Join[myList, {3, 4}]      (* {1, 2, 3, 4, 3, 4} *)
+myList[[2]] = 5          (* {1, 5, 3, 4} - 这确实更新了myList *)
+
+(* 关联，又称字典/哈希值 *)
+myHash = <|"Green" -> 2, "Red" -> 1|>   (* 创建一个关联 *)
+myHash[["Green"]]                       (* 2, 用它 *)
+myHash[["Green"]] := 5                  (* 5, 更新它 *)
+myHash[["Puce"]] := 3.5                 (* 3.5, 拓展它 *)
```
Comment (imba-tjd) [inline zh-cn/wolfram-cn.html.markdown:127]:
```suggestion
myHash[["Green"]]                       (* 2, 使用 *)
myHash[["Green"]] := 5                  (* 5, 更新 *)
myHash[["Puce"]] := 3.5                 (* 3.5, 扩展 *)
```

File: zh-cn/wolfram-cn.html.markdown [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+MyFirst[]                                    (* 你好，世界 *)
+
+(* C-Style 的 For 循环，注：在 Mathematica 中使用 For 循环是低效的 *)
+PrintTo[x_] := For[y=0, y<x, y++, (Print[y])]  (* 开始，测试，自增，循环体 *)
+PrintTo[5]                                     (* 0 1 2 3 4 *)
+
+(* While 循环 *)
+x = 0; While[x < 2, (Print@x; x++)]     (* 带测试和程序体的 while 循环 *)
+
+(* 如果 和 条件 *)
```
Comment (imba-tjd) [inline zh-cn/wolfram-cn.html.markdown:80]:
```suggestion
(* If 和 条件 *)
```

File: zh-cn/wolfram-cn.html.markdown [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+x = 8; If[x==8, Print@"Yes", Print@"No"]   (* 条件，返回真的情况下，否则 *)
```
Comment (imba-tjd) [inline zh-cn/wolfram-cn.html.markdown:81]: 我觉得还不如不翻

**Review by imba-tjd — CHANGES_REQUESTED** [review]

File: zh-cn/wolfram-cn.html.markdown (hunk 5 above)
Comment (wuyudi) [reply zh-cn/wolfram-cn.html.markdown]: fixed

**Review by imba-tjd — APPROVED** [review]

**Comment by imba-tjd** [comment]
Hi，方便把后五条提交rebase成一条然后force push吗？如果不行也没关系。

**Comment by wuyudi** [comment]
solved.

**Comment by imba-tjd** [comment]
OK, Thank you!

**Pull request merged**